composer.json

```diff
@@ -10,7 +10,8 @@
     ],
     "homepage": "https://bitbucket.org/digipolisgent/drupal_theme_gent-base",
     "require": {
-        "php": "^8.1"
+        "php": "^8.1",
+        "drupal/twig_typography": "^2.1"
```

Comment:
This breaks certain builds like travis of drupal projects using this theme because drupal core gets loaded (a second time) in the base_theme's vendor dir.